### ISSUE TITLE
Hello, is there any article description for this program

### ISSUE THREAD

**Issue by zhang494** [GiulioRusso/5G-power-optimization#1]

**Comment by GiulioRusso** [comment]
Yes, my thesis, but it’s not published here. Write to my mail giulio@elioecaterina.it if you want to know more

**Comment by Hiefun** [comment]
Hello, has your article been published?

**Comment by GiulioRusso** [comment]
No
Da: "Hiefun" ***@***.***
A: "GiulioRusso/5G-power-optimization" ***@***.***
Cc: "Giulio Russo" ***@***.***,"Comment" ***@***.***
Data: Fri, 02 Dec 2022 18:54:52 -0800
Oggetto: Re: [GiulioRusso/5G-power-optimization] Hello, is there any article description for this program (#1)


Hello, has your article been published?

**Comment by Hiefun** [comment]
thank you



---Original---
From: "Giulio ***@***.***&gt;
Date: Sat, Dec 3, 2022 13:33 PM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [GiulioRusso/5G-power-optimization] Hello, is there any articledescription for this program (#1)




 No
 Da: "Hiefun" ***@***.***
 A: "GiulioRusso/5G-power-optimization" ***@***.***
 Cc: "Giulio Russo" ***@***.***,"Comment" ***@***.***
 Data: Fri, 02 Dec 2022 18:54:52 -0800
 Oggetto: Re: [GiulioRusso/5G-power-optimization] Hello, is there any article description for this program (#1)
 
 
 Hello, has your article been published?

**Comment by 520cwj** [comment]
Hello, has your article been published? Could I get this paper? If so,please send it to my email 1627116769@qq.com,thank you very much!

**Comment by GiulioRusso** [comment]
I have to talk with my supervisor, since this is a bachelor thesis
